src/pybind/mgr/orchestrator/module.py

```diff
@@ -1187,6 +1189,52 @@ def _apply_iscsi(self,
 
         return self._apply_misc([spec], dry_run, format, no_overwrite)
 
+    @_cli_write_command('orch apply snmp-gateway')
+    def _apply_snmp_gateway(self,
+                            snmp_version: SNMPVersion,
+                            destination: str,
+                            port: Optional[int] = 9464,
```

Comment:
Le't avoid putting the default value in here. Can we instead default to None here?